### DISCUSSION TITLE
Translation Swedish

### DISCUSSION THREAD

**Discussion by Hi-there-how-are-u** [sircharlo/meeting-media-manager#259] [Translations]
Hello, I am willing to translate the program to Swedish.
I have already downloaded the en.json and I am almost done, but I do not understand what is Target meeting?
When I am done I will upload the file here since forking is not my cup of tea... :)

Also is there any way I can test the file before uploading? It would be great to see the text in it's context.
Have a great day!

/Marko

**Comment by sircharlo** [comment]
That's great to hear, thanks! I'll be sure to add that new translation for the next release.
"Target meeting" is a holdover from a long time ago; I'll change the string title to be less confusing. It's basically "meeting" or "select a meeting".

**Reply by sircharlo** [reply]
Also, no unfortunately there's no easy way to test the new strings, unless you clone the repo and run the app using `npm i; node run start`

**Comment by Hi-there-how-are-u** [comment]
Thank you very much for explaining the meaning.
Here is the file, maybe I need to just adjust it later if something is wrong...
Have a great day
[sv.json.txt](https://github.com/sircharlo/jw-meeting-media-fetcher/files/8504994/sv.json.txt)
!

**Reply by sircharlo** [reply]
Thank you so much! Committed, will be released with next version.

**Comment by Hi-there-how-are-u** [comment]
A question, how do I know what to translate when you update the program?
I noticed atleast two new things to translate:  Disable hardware acceleration and enable OBS studio compatibility.
But is that all or is there something else to translate?
I hoped that the english would be in the sv.json but that was not the case.
https://raw.githubusercontent.com/sircharlo/jw-meeting-media-fetcher/master/locales/sv.json
